### PR TITLE
feature: transpile es6 --> es5 and sass --> css

### LINKED ISSUE #44: Packages should be transpiled to ES5
Currently we don't have a build step for the packages, which requires npm users to transpile the packages themselves. The packages should be ES5 for everyone to consume.

### PR DESCRIPTION
**Pull request by moog16**
fixes #44 and #45 

Reviewer should be able to run

```
. ./scripts/release/pre-release.sh
```

That will run the build, copy the scripts into packages, and run the tests.

- To run build: `npm run build`. This will build all the ES5 and css files.
- To copy all the scripts into the `dist` directories into each package run: 
`node ./scripts/release/cp-pkg.js`.

### CODE REVIEW

File: packages/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const chunks = [
```
Comment (moog16) [inline packages/webpack.config.js]: I could use fs to read all directories and automatically pull in packages.

File: .gitignore [hunk 2]
```diff
@@ -8,3 +8,5 @@ node_modules
 key.json
 *.golden.png
 lerna-debug.log
+build/**
```
Comment (kfranqueiro) [inline .gitignore]: Shouldn't just `build` and `dist` be enough here?

Comment (moog16) [reply .gitignore]: yup you're right. I added that when they were already built. so I needed the `**`, but if you delete the directories and just have `build` and `dist` it works fine.

File: docs/guidelines.md [hunk 3]
```diff
@@ -23,3 +23,34 @@ class Dog extends React.Component {
   }
 };
 ```
+
+## Importing with ES5
+
+The NPM packages all contain a `/dist` directory, which includes:
```
Comment (williamernest) [inline docs/guidelines.md]: all -> each

File: scripts/release/cp-pkgs.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (williamernest) [inline scripts/release/cp-pkgs.js]: copyright year -> 2018

File: packages/webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
+const webpack = require('webpack');
```
Comment (williamernest) [inline packages/webpack.config.js]: file license

File: scripts/release/pre-release.sh [hunk 6]
```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+##
+# Copyright 2016 Google Inc. All Rights Reserved.
```
Comment (williamernest) [inline scripts/release/pre-release.sh]: year -> 2018

File: test/screenshot/screenshot.js [hunk 7]
```diff
@@ -201,7 +201,7 @@ export default class Screenshot {
   async takeScreenshot_() {
     const browser = await puppeteer.launch();
     const page = await browser.newPage();
-    await page.goto(`http://localhost:8080/${this.urlPath_}`, {'waitUntil': 'networkidle2'});
+    await page.goto(`http://localhost:8080/${this.urlPath_}`, {'waitUntil': 'networkidle2', 'timeout': 90000});
```
Comment (williamernest) [inline test/screenshot/screenshot.js]: Should each screenshot page have a 90 second timeout?

**Pull request merged**